mlrun/api/crud/model_endpoints.py

```diff
@@ -519,25 +519,29 @@ def deploy_monitoring_functions(
         model_monitoring_access_key: str,
         db_session: sqlalchemy.orm.Session,
         auth_info: mlrun.api.schemas.AuthInfo,
+        tracking_policy: mlrun.utils.model_monitoring.TrackingPolicy,
```

Comment:
The whole goal here is eventually called model **monitoring**, so shouldn't it be called `monitoring_configurations` or `monitoring_policy`, and the method `set_monitoring_configurations/policy` instead of `set_tracking`. I know its not your terminology but perhaps it can be changed?

I mean, even the doc string of this attribute is "Model monitoring configurations." :)